idd/Energy+.idd.in

```diff
@@ -34671,7 +34671,6 @@ ZoneHVAC:WaterToAirHeatPump,
         \object-list HeatingCoilName
         \note  Needs to match in the supplemental heating coil object
    N11, \field Maximum Supply Air Temperature from Supplemental Heater
-        \required-field
```

Comment:
Why is this object being touched here? If you want to make this not required, then it should be \default autosize

Reply:
I wanted it to be an optional field, no other reason. OK. will add " \required-field" and default to autosize. @mjwitte Thanks.

Reply:
@mjwitte never mind, I thought this was the VF terminal unit. It must be accidental change. will fix both.